### ISSUE TITLE
Send animated stickers as gif instead of png

### ISSUE THREAD

**Issue by Neelgodly** [Aliucord/plugin-requests#718]
### Discord Account

Jiggle-Wiggle#0726

### Plugin Description

Using the fakestickers plugin, stationary stickers work totally fine. Animated though are sent as apng which works for the sender with 'AnimatedApng' plugin but won't work for regular discord users. It would be cool if it could be sent as a gif instead like betterdiscord's free stickers plugin does. 

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by ItzOnlyAnimal** [comment]
stickers are animated png/Lottie so not possible without converting to gif and increasing file size

**Comment by Neelgodly** [comment]
Yeah i knew stickers wer stored as apng but the fact that a BD plugin could do it made me feel like it would be possible with aliucord as well.

Anyways, thanks for the reply!

**Comment by ItzOnlyAnimal** [comment]
Well the BD plugin probably converts it to GIF anyways, so yes possible but not practical for mobile plugin

**Comment by Neelgodly** [comment]
Okay, I know this is late but I literally saw this just now... Can you please explain why it isn't practical for mobile? Thank you!

**Comment by C10udburst** [comment]
because the betterdiscord plugin uses wasm to convert apng to gif

**Comment by Mythical-Forest-Collective** [comment]
Could use asmble to compile WASM to Java :trollface:
